certbot/compat/misc.py

```diff
@@ -17,6 +17,19 @@
 from certbot import errors
 from certbot.compat import os
 
+
+# UMASK_FOR_PRIVATE_KEY_PERMISSIONS defines what are the permissions flags to keep
```

Comment:
nit: Is this really a umask for any definition of umask given at https://en.wikipedia.org/wiki/Umask? I think we should just change this and all other references to `MASK_FOR_PRIVATE_KEY_PERMISSIONS`.

Reply:
Done.